pyanaconda/network.py

```diff
@@ -258,6 +258,12 @@ def initialize_network():
     log.debug("Devices found: %s",
               [dev.device_name for dev in get_supported_devices()])
 
+    if util.is_stage2_on_nfs() and network_proxy.Kickstarted:
+        msg = "Using kickstart network configuration with installer image (stage2) provided " \
+            "via nfs server can freeze the installation."
+        log.warning(msg)
+        print("WARNING:", msg)
```

Comment:
Is this forgetten debug log? Why we need log.warning and print?

Reply:
I think we want both print - to show the warning in the UI (on the tty1), and log - to have the message in the logs. The goal is to make it discoverable when hitting the situation of the freeze.

I was playing also with `anaconda.loggers.get_stdout_logger` that goes both on the screen and into anaconda.log but it did not look as I wanted (warning enough) on the tty1.
![Screenshot from 2024-10-24 11-03-01](https://github.com/user-attachments/assets/4a4759f4-a848-4962-9dee-04941a97660d)